ConanInstall.cmake

```
@@ -57,29 +71,30 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   conan_cmake_run(REQUIRES
     ${CONAN_READLINE}
     ${CONAN_QT}
-    openssl/1.1.0l # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
+    "openssl/1.1.0l#7f3fa5cfcfba31fffa344c71a9795176" # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
     # Track NREL/stable in general, on a feature branch this could be temporarily switched to NREL/testing
-    openstudio_ruby/2.5.5@nrel/stable    # TODO: Temp #@nrel/stable
-    boost/1.73.0
-    pugixml/1.10
-    jsoncpp/1.9.3
-    zlib/1.2.11
-    fmt/7.0.1
-    sqlite3/3.32.3
-    cpprestsdk/2.10.16
-    websocketpp/0.8.2
-    geographiclib/1.50.1
-    swig/4.0.2
+    "openstudio_ruby/2.5.5@nrel/stable#dcdbeda50b48bbd0b2bf3dd1612b3896"    # TODO: Temp #@nrel/stable
+    "boost/1.73.0#4129a76c9b83c300fc103e36d1908792"
+    "pugixml/1.10#64b3ebc897bb9d9854c8a2443bf112a8"
+    "jsoncpp/1.9.3#073a6d3cb40911d7c8027bddb6ae7dbf"
+    "zlib/1.2.11#0df31fd24179543f5720ec7beb2a88d7"
+    "fmt/7.0.1#0580b1492b1dddb43b1768e68f25c43c"
+    "sqlite3/3.32.3#914492672c458f8be511e3800c14c717"
+    "cpprestsdk/2.10.16#d097ff9a8719d9d0ed34293c2ebd90ed"
+    "websocketpp/0.8.2#6d77b9b8a2368fa5fd5377af0c0ca211"
+    "geographiclib/1.50.1#b1a7966385dead17ec170b25a99cf71b"
+    "swig/4.0.2#bfafb16cd2bea6af3b8003163abcbd09"
     ${CONAN_GTEST}
 
     # Override to avoid dependency mismatches
-    bzip2/1.0.8
+    #"bzip2/1.0.8#d4a5c7144832d75fc3f349c5346160b0"
+    #"libyaml/0.2.5#9e234874df88c3ba7249f6d1368fceaf"
     BASIC_SETUP CMAKE_TARGETS NO_OUTPUT_DIRS
     OPTIONS ${CONAN_OPTIONS}
     BUILD ${CONAN_BUILD}
     # Passes `-u, --update`    to conan install: Check updates exist from upstream remotes
     # That and build=outdated should ensure we track the right
-    UPDATE
+    # UPDATE
```

```suggestion
    UPDATE
```

Actually it seems that UPDATE is needed to go check on the remote to see if we can find the right revisions in case we have some revisions already and they don't match. Too bad.